__tests__/comments.test.js

```diff
@@ -50,3 +50,70 @@ describe("GET comments", () => {
       });
   });
 });
+
+describe("POST comments", () => {
+  test("receive 201 when POST /api/articles/:article_id/comments ", () => {
+    const testComment = {
+      username: "butter_bridge",
+      body: "Oh, I've got compassion running out of my nose, pal! I'm the Sultan of Sentiment!",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(testComment)
+      .expect(201);
+  });
```

Comment:
Please add a test 201 where extra properties are ignored and only the relevant ones are considered when posting. (e.g. {username: "butter_bridge", body: "hello", toBeIgnored: "something"})